### PR TITLE
Fix fast properties

### PR DESCRIPTION
**Pull request by sargun**
This is a fix for properties which are dynamically loaded

### CODE REVIEW

File: properties/properties.go [hunk 1]
```diff
@@ -60,11 +60,14 @@ func (qis *QuiteliteInputSource) getVal(name string) (interface{}, bool) {
 func (qis *QuiteliteInputSource) Int(name string) (int, error) {
 	otherGenericValue, exists := qis.getVal(name)
 	if exists {
-		otherValue, isType := otherGenericValue.(int)
+		otherValue, isType := otherGenericValue.(float64)
```
Comment (rickkw) [inline properties/properties.go]: Does quitelite treat numbers, including integer, as float64?  Can its implementation change in the future that returns int for integer value?

File: properties/properties.go [hunk 2]
```diff
@@ -45,7 +39,7 @@ func fetchQuiteLiteSource(alternateURIFlag string) (altsrc.InputSourceContext, e
 
 	}()
 	ret := QuiteliteInputSource{}
-	err = json.NewDecoder(resp.Body).Decode(&ret)
+	err = json.NewDecoder(resp.Body).Decode(&ret.valueMap)
```
Comment (rickkw) [inline properties/properties.go]: is this the main bug?

Comment (sargun) [reply properties/properties.go]: There were a few bugs:
1. This one
2. There was a problem with conversions of floats / ints.
3. The actual function wasn't being called (https://github.com/Netflix/titus-executor/pull/161/files#diff-7cae71039937aae66ba05a7cdcf970fcR75)
4. The flags needed to be enabled with altsrc. https://github.com/Netflix/titus-executor/pull/161/files#diff-587d2988572f886f6e2554c7e0ea6ba7R229

File: properties/properties.go (hunk 1 above)
Comment (sargun) [reply properties/properties.go]: Ye
Yes, see here: https://golang.org/pkg/encoding/json/

> To unmarshal JSON into an interface value, Unmarshal stores one of these in the interface value:

```
bool, for JSON booleans
float64, for JSON numbers
string, for JSON strings
[]interface{}, for JSON arrays
map[string]interface{}, for JSON objects
nil for JSON null
```

**Pull request merged**